examples/task-tracker/client/tasks/add_comment_to_task_parameters.go

```diff
@@ -112,6 +133,16 @@ func (o *AddCommentToTaskParams) SetID(id int64) {
 	o.ID = id
 }
 
+// Validate these params
+func (o *AddCommentToTaskParams) Validate(formats strfmt.Registry) error {
+
+	if err := o.Body.Validate(formats); err != nil {
```

Comment:
so are you saying, @casualjim, that the reason it is not rendering is an unreported bug?